install_files/php/Installer.php

```diff
@@ -111,6 +111,9 @@ protected function onCheckRequirement()
             case 'allowUrlFopenConfig':
                 $result = ini_get('allow_url_fopen');
                 break;
+            case 'urlRewriting':
+                $content = @file_get_contents($this->getBaseUrl() . 'rewrite/test/');
+                $result = ($content === 'Rewrite successful');
```

Comment:
Shouldn't this have a break statement here too?

Reply:
@LukeTowers for code cleanliness, yes, but as its technically the last case, it doesn't *require* it.

Reply:
Of course :) I'm all about that code cleanliness though, same principle as trailing commas